src/Console/Application.php

```diff
@@ -218,4 +223,19 @@ protected function initializeComposerHelper()
 
         return $this->composerHelper = new Helper\ComposerHelper($configuration, $rootPackage);
     }
+
+    /**
+     * Prefixes the cwd to the path given by the user
+     *
+     * @param string $configPath
+     * @return string
```

Comment:
Not sure if an enter is required above '@return'. If not, just ignore this

Reply:
It is needed.